src/components/views/rooms/RoomHeader.tsx

```diff
@@ -106,6 +106,8 @@ export default function RoomHeader({ room }: { room: Room }): JSX.Element {
     }, [room, directRoomsList]);
     const e2eStatus = useEncryptionStatus(client, room);
 
+    const notificationsEnabled = useFeatureEnabled("feature_notifications");
```

Comment:
The setting uses `ReloadOnChangeController`. `useFeatureEnabled` adds a watcher. This is not required here. What do you think about tweaking the `useFeatureEnabled` hook with an option whether to watch the setting?